CHANGES.md

```diff
@@ -9,6 +9,9 @@
   You can now query for documents without the existence of any indexes.
 - [NEW] New fields `documentsReplicated` and `batchesReplicated` added
   to ReplicationCompleted class
+- [DEPRECATED] Deprecated the  following classes: ReplicatorFactory, Replication, 
+  PullReplication and PushReplication.
+- [NEW] New ReplicatorBuilder API, this should be used to create replications in the future. 
```

Comment:
I think you should give a quick example of how to move from the deprecated API to the new API here, covering the simple `oneway` case.